providers/certification-client/units/client-cert-iot-ubuntucore-24.pxu

```diff
@@ -77,6 +77,9 @@ nested_part:
   after-suspend-wireless-manual
   after-suspend-wireless-wifi-master-mode-manual
   after-suspend-wwan-manual
+exclude:
+  bluetooth4/HOGP-mouse
+  after-suspend-bluetooth4/HOGP-mouse
```

Comment:
Why we don't need to test the BT mouse? I suggest that if we are nesting the same plan as PC nesting, we could retain the same coverage.

Reply:
Core and Server images don't have GUI by default, it means you cannot see the cursor even the BT mouse be connected.